tests/consequence_prediction/vep_mapping_pipeline/test_consequence_mapping.py

```diff
@@ -0,0 +1,81 @@
+from consequence_prediction.vep_mapping_pipeline.consequence_mapping import extract_consequences
+
+
+def test_extract_consequences():
+    vep_results = [
+        {
+            'input': '10 27169969 . C A',
+            'transcript_consequences': [
+                {
+                    'gene_symbol': 'MASTL',
+                    'gene_id': 'ENSG00000120539',
+                    'biotype': 'protein_coding',
+                    'consequence_terms': ['missense_variant']
+                },
+                # {  # missing gene symbol is okay
+                #     'gene_id': 'ENSG00000120538',
+                #     'biotype': 'protein_coding',
+                #     'consequence_terms': ['missense_variant']
+                # },
+                {  # less severe consequences should be filtered out
```

Comment:
Now that #225 is merged, I'm guessing this could be enabled back?

Reply:
Indeed, I'll rebase and enable this.